content/docs/command-reference/remote/add.md

```diff
@@ -307,16 +307,11 @@ $ dvc remote add -d myremote ssh://user@example.com/path/to/dir
 
 > See also `dvc remote modify` for a full list of SSH parameters.
 
-⚠️ DVC requires both SSH and SFTP access to work with SSH remote storage. Please
-check that you are able to connect both ways to the remote location, with tools
-like `ssh` and `sftp` (GNU/Linux).
-
-> Note that your server's SFTP root might differ from its physical root (`/`).
-> (On Linux, see the `ChrootDirectory` setting in `/etc/ssh/sshd_config`.) In
-> these cases, the path component in the SSH URL (e.g. `/path/to/dir` above)
-> should be specified relative to the SFTP root instead. For example, on some
-> Sinology NAS drives, the SFTP root might be in directory `/volume1`, in which
-> case you should use path `/path/to/dir` instead of `/volume1/path/to/dir`.
```

Comment:
I decided to remove these details. They seem more like tips unrelated to DVC. Users can research SFTP details on other sites easily, I think. Left a quick note: "Note that the server's SFTP root might differ from its physical root (`/`)." that should be enough of a tip for the few users running into that issue.

Reply:
ounds good! and we can elaborate when we have a separate page per remote I guess

Reply:
We're planning to do separate pages per remote? I don't recall that decision. It sounds like a good plan except that the remote types are already burried in docs->cmd ref->remote->add/modify->expandable sections so not sure adding yet another level to that tree is a good idea.

Maybe the remote types should all be in the remote ref. index? And the remote concept explanation in Basic Concepts.